nuwebots/controllers/nugus_controller/CMakeLists.txt

```diff
@@ -17,9 +19,13 @@ target_include_directories(nugus_controller SYSTEM PRIVATE "${PROJECT_BINARY_DIR
 
 # Use webots, Eigen and protobuf libraries
 target_link_libraries(nugus_controller PRIVATE webots::webots)
+target_link_libraries(nugus_controller PRIVATE "${WEBOTS_HOME}/lib/controller/libCppController.so")
 target_link_libraries(nugus_controller PRIVATE Eigen3::Eigen)
+# Link against the protobuf-lite target (this will also add any necessary include directories to our target)
 target_link_libraries(nugus_controller PRIVATE protobuf::libprotobuf-lite)
 
+target_link_libraries(nugus_controller PRIVATE "${WEBOTS_HOME}/lib/controller/libCppController.so")
+
```

Comment:
```suggestion
target_link_libraries(nugus_controller PRIVATE webots::webots)
target_link_libraries(nugus_controller PRIVATE Eigen3::Eigen)
# Link against the protobuf-lite target (this will also add any necessary include directories to our target)
target_link_libraries(nugus_controller PRIVATE protobuf::libprotobuf-lite)

```